packages/lab/src/toolbar/ToolbarButton.css

```diff
@@ -5,4 +5,5 @@
 .saltToolbarButton:not([data-is-inside-panel]) {
   --saltButton-fontSize: 0;
   --saltButton-letterSpacing: 0;
+  gap: 0;
```

Comment:
This is needed for when the button is collapsed and the text is hidden

Reply:
would this need a changeset?

Reply:
Not really, it's required due to the Button changes so that ToolbarButton doesn't break in its collapsed state.